$Id$

	evil kludge because configure only checks the host system,
	not the build system, yet builds a tool to run on it (cf_gen)

	the real way to do this is probably to enhance configure.in
	the way gcc uses it if doing a build with build!=(host==target),
	but this should be enough for now; contact tg@ (PR#71) if it isn't

--- squid-2.5.STABLE13.orig/include/config.h	2005-04-23 03:32:27.000000000 +0200
+++ squid-2.5.STABLE13/include/config.h	2006-09-18 19:21:26.000000000 +0200
@@ -34,7 +34,36 @@
 #ifndef SQUID_CONFIG_H
 #define SQUID_CONFIG_H
 
+#ifndef BUILD_HOST_TOOL
 #include "autoconf.h"		/* For GNU autoconf variables */
+#else
+/* we just hope our host systems can cope */
+#define STDC_HEADERS 1
+#define HAVE_MEMCPY 1
+#define HAVE_MEMMOVE 1
+#define HAVE_MEMSET 1
+#define HAVE_MKSTEMP 1
+#define HAVE_MKTIME 1
+#define HAVE_ASSERT_H 1
+#define HAVE_CTYPE_H 1
+#define HAVE_DIRENT_H 1
+#define HAVE_ERRNO_H 1
+#define HAVE_FCNTL_H 1
+#define HAVE_GETOPT_H 1
+#define HAVE_INTTYPES_H 1
+#define HAVE_LIMITS_H 1
+#define HAVE_SIGNAL_H 1
+#define HAVE_STDARG_H 1
+#define HAVE_STDDEF_H 1
+#define HAVE_STDIO_H 1
+#define HAVE_STDLIB_H 1
+#define HAVE_STRING_H 1
+#define HAVE_SYS_PARAM_H 1
+#define HAVE_SYS_STAT_H 1
+#define HAVE_SYS_TIME_H 1
+#define HAVE_TIME_H 1
+#define HAVE_UNISTD_H 1
+#endif
 #include "version.h"
 
 /****************************************************************************
